app/views/merchant/items/index.html.erb

```diff
@@ -1,5 +1,23 @@
-<h1>My Items</h1>
+<h1>My Items - <%= link_to current_user.merchant.name, merchant_path(current_user.merchant.id) %></h1>
+<p align="center"><%= link_to "Add New Item", "/merchant/#{current_user.merchant.id}/items/new" %></p>
 
-<% @items.each do |item| %>
-  <%= link_to item.name, "/merchant/items/#{item.id}" %>
-<% end %>
+<section class="grid-container">
+  <% @items.each do |item| %>
+    <section class = "grid-item" id= 'item-<%=item.id%>'>
+      <%= render partial: '/partials/item_info', locals: {item: item, path: merchant_item_path(item.id)}%>
```

Comment:
This is exactly what you were explaining when we were talking about sending in a path as a local, correct? Love it.